sormas-ui/src/main/java/de/symeda/sormas/ui/statistics/StatisticsView.java

```diff
@@ -726,22 +731,12 @@ public void generateChart() {
 		}
 		hcjs.append("],");
 
-		hcjs.append("exporting: {\n" +
```

Comment:
@DavidBaldsiefen Didn't notice this before. Please add a tag to keep the original formatting, because it is much easier to read.